### PR TITLE
improve Change Password modal UI

### PR DESCRIPTION
**Pull request by faissaloux**
Hello 👋🏻

In this PR I have tried to improve the Change Password modal UI

# pmahomme
## Before
Screenshot 2022-05-26 141840

## After
Screenshot 2022-05-26 143436

# Original
## Before
Screenshot 2022-05-26 141853

## After
Screenshot 2022-05-26 143507

# Metro
## Before
Screenshot 2022-05-26 141906

## After
Screenshot 2022-05-26 143534

# Bootstrap
## Before
Screenshot 2022-05-26 141920

## After
Screenshot 2022-05-26 143559

Signed-off-by: Faissal Wahabali <fwahabali@gmail.com>

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
This is very good work
I am grateful for the screenshots, this is very cool!!
You need to run a `yarn run js-lint --fix` and this will be okay

**Comment by ibennetch** [comment]
I also love this improvement.

The screenshots really help show the change.

Thank you for this.

File: test/classes/Server/PrivilegesTest.php [hunk 1]
```diff
@@ -1838,11 +1838,6 @@ public function testGetFormForChangePassword(): void
             $html
         );
 
-        //labels
-        $this->assertStringContainsString(
```
Comment (williamdes) [inline test/classes/Server/PrivilegesTest.php:1842]: Can another string be used to assert?

File: test/classes/Server/PrivilegesTest.php [hunk 2]
```diff
@@ -1840,7 +1840,7 @@ public function testGetFormForChangePassword(): void
 
         //labels
         $this->assertStringContainsString(
-            __('Change password'),
+            __('change_password_form'),
```
Comment (williamdes) [inline test/classes/Server/PrivilegesTest.php:1843]:
```suggestion
            'change_password_form',
```
It has no translation ;)

Comment (faissaloux) [reply test/classes/Server/PrivilegesTest.php]: Yeah 😅

**Review by williamdes — APPROVED** [review]
💯

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**